src/renderers/shaders/ShaderChunk/lights_pars_begin.glsl.js

```diff
@@ -158,7 +163,10 @@ float getSpotAttenuation( const in float coneCosine, const in float penumbraCosi
 		float angleCos = dot( light.direction, spotLight.direction );
 
 		float spotAttenuation = getSpotAttenuation( spotLight.coneCos, spotLight.penumbraCos, angleCos );
-
+		if (spotLight.projector) {
+			vec3 spotLightCoord = spotCoord.xyz / spotCoord.w;
+			spotAttenuation = clamp((-2.0 * sdBox(spotLightCoord.xy - vec2(0.5), vec2(0.5, 0.5))) * (-1.0 / ((1.0 - acos(spotLight.penumbraCos)) - 1.0)), 0.0, 1.0);
+		}
```

Comment:
Can we update this to use the three.js code style and remove the unnecessary parentheses.

Reply:
@N8python See the guide here, which includes tools for automatically formatting the code: 

https://github.com/mrdoob/three.js/wiki/Mr.doob's-Code-Style%E2%84%A2